docs/operators/metrics.md

```diff
@@ -81,12 +81,19 @@ in production systems.
 
 The record `system` has the following schema:
 
-| Field                | Type    | Description                                     |
-| :------------------- | :------ | :---------------------------------------------- |
-| `running_actors`     | `int64` | Number of currently running actors.             |
-| `queued_messages`    | `int64` | Number of messages waiting in actor mailboxes.  |
-| `processed_messages` | `int64` | Number of messages processed since last metric. |
-| `rejected_messages`  | `int64` | Number of messages rejected since last metric.  |
+| Field                    | Type     | Description                                                         |
+| :----------------------- | :------- | :------------------------------------------------------------------ |
+| `running_actors`         | `int64`  | Number of currently running actors.                                 |
+| `all_messages`           | `record` | Information about the total message metrics                         |
+| `messages_without_actor` | `record` | Information about the message metrics not associated with an actor. |
+| `messages_by_actor`      | `record` | Information about the message metrics, keyed actor name.            |
```

Comment:
Let's make this a list instead, with the actor name as a field. Could make actor name `null` then. Then we only have:

- `running_actors`
- `all_messages`
- `messages_by_actor`